### PR TITLE
es: CR2 Day 2 translations

### PR DESCRIPTION
**Pull request by henrif75**
Professional translations for CR2 content.
Changes are being staged in a separate branch until all Spanish content is reviewed.

#1463 #284

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -7478,12 +7659,19 @@ msgid ""
 "you can (i.e. you call it once), or `FnMut` else, and last `Fn`. This allows "
 "the most flexibility for the caller."
 msgstr ""
+"Cuando defines una función que lleva un cierre, debes usar `FnOnce` si es "
```
Comment (deavid) [inline po/es.po:7662]:
```suggestion
"Cuando defines una función que admite un closure, debes usar `FnOnce` si es "
```

**Review by deavid — CHANGES_REQUESTED** [review]

File: po/es.po [hunk 2]
```diff
@@ -5133,79 +5133,85 @@ msgstr ""
...
 "minutes)"
 msgstr ""
+"[Ejercicio: evaluación de expresiones](./pattern-matching/exercise.md) (30 "
+"minutos)"
 
 #: src/pattern-matching/destructuring.md
 msgid "Like tuples, structs and enums can also be destructured by matching:"
 msgstr ""
+"Al igual que las tuplas, las estructuras y las enumeraciones también se "
+"pueden desestructurar con la coincidencia:"
```
[the remark this answers is not in the document]
Comment (henrif75) [reply po/es.po:5199]: Sounds good!

File: po/es.po [hunk 3]
```diff
@@ -7220,25 +7377,31 @@ msgid ""
...
 
 #: src/std-traits/casting.md
 msgid ""
 "The results of `as` are _always_ defined in Rust and consistent across "
 "platforms. This might not match your intuition for changing sign or casting "
 "to a smaller type -- check the docs, and comment for clarity."
 msgstr ""
+"Los resultados de `as` se definen _siempre_ en Rust y son coherentes en "
+"todas las plataformas. Es posible que no coincida con tu idea de cambiar el "
+"signo o convertirlo a otro de menor tamaño. Consulta los documentos y añade "
```
[the remark this answers is not in the document]
Comment (henrif75) [reply po/es.po:7404]: I think so, to speak up during the class. Would you suggest a better translation?

[the remark this answers is not in the document]
Comment (deavid) [reply po/es.po:7403]: "Consulta los documentos y/o pregunta si tienes cualquier duda."

I doubt there's a good way to make the translation closer to the original.

**Comment by deavid** [comment]
As I answered to one of your comments, I'm holding off review until I get pinged again. If you don't push any further change and want review, please ping me here, for example with a comment. Otherwise I probably will forget to check this.

File: po/es.po (hunk 3 above)
[the remark this answers is not in the document]
Comment (henrif75) [reply po/es.po]: Thanks @deavid I made the change.

**Comment by henrif75** [comment]
> As I answered to one of your comments, I'm holding off review until I get pinged again. If you don't push any further change and want review, please ping me here, for example with a comment. Otherwise I probably will forget to check this.

Hi @deavid All the suggestions were committed.

**Review by deavid — APPROVED** [review]

**Comment by henrif75** [comment]
@deavid Thanks again!

**Pull request merged**